Cargo.toml

```diff
@@ -111,3 +111,16 @@ features = ["ansi", "env-filter", "fmt", "time"]
 lto = 'thin'
 panic = 'abort'
 codegen-units = 1
+
+[patch.crates-io]
+bridgetree = { git = "https://github.com/zcash/incrementalmerkletree", rev = "ffe4234788fd22662b937ba7c6ea01535fcc1293" }
+incrementalmerkletree = { git = "https://github.com/zcash/incrementalmerkletree", rev = "ffe4234788fd22662b937ba7c6ea01535fcc1293" }
```

Comment:
Since these two crates have releases, we should be able to just depend directly on those released versions.

Reply:
I kept this identical to what the revision of `zcash/librustzcash` depends on, to ensure compatibility.